components/event-bus/Makefile

```diff
@@ -29,15 +30,14 @@ verify:: all
 # base import path
 PKG := github.com/kyma-project/kyma/components/event-bus
 # produce goals for each cmd program ("bin1 bin2 ...")
-cmds := $(foreach cmd,$(shell go list -f '{{ if eq .Name "main" }}{{ .ImportPath }}{{end}}' ./...),$(notdir $(cmd)))
+#cmds := $(foreach cmd,$(shell go list -f '{{ if eq .Name "main" }}{{ .ImportPath }}{{end}}' ./...),$(notdir $(cmd)))
```

Comment:
Is the leftover comment on purpose?

Reply:
left in just in case we figure out a way to have this evaluated either: 
- using docker and not slow down execution of the makefile
- using go on the machine that executes the Makefile